src/themes/teams/components/ItemLayout/itemLayoutStyles.ts

```diff
@@ -0,0 +1,55 @@
+import { pxToRem } from '../../../../lib'
+import { debugRoot } from '../../../../styles/debugStyles'
+
+const itemLayoutStyles = {
+  root: ({ props, variables }) => {
+    const { debugLayout, important } = props
+    return {
+      ...(debugLayout && debugRoot()),
+      gridTemplateRows: `minmax(${variables.itemHeight}, max-content)`,
+      paddingLeft: variables.itemPaddingLeft,
+      paddingRight: variables.itemPaddingRight,
+
+      ...(important && {
+        fontWeight: 'bold',
+      }),
+    }
+  },
+
+  media: ({ props }) => {
+    const { important } = props
```

Comment:
Since `important` will likely vary depending on the parent component implementing the items, let's move this prop and style also back to the ListItem side.  Keeping just layout and DOM structure in the layouts.